src/org/sparkboard/server/server.clj

```diff
@@ -13,9 +206,9 @@
     (.stop @server)))
 
 (def routes
-  ;; TODO provide separate URLs where Slack allows, so we can pass to
-  ;; individual handlers here rather than by examining the request
-  ["/" handle/incoming])
+  ["/" {"slack-api" (fn [x] (#'incoming x))
+        "slack-api/oauth-redirect" #'slack-oauth/redirect
+        "slack/install" #'slack-oauth/install-redirect}])
```

Comment:
I was having issues with having to restart the server every time I re-evaluated a handler in order for changes to be picked up

Reply:
Also, forgot to explicitly mention that I changed the main “incoming” link to be `/slack-api`, which requires updating slack settings. Important because we will certainly use the root path for something else in the future.

Reply:
>having to restart the server every time I re-evaluated a handler

Interesting — normally `#'app` in `run-jetty` has been sufficient, but that's definitely a good reason.